package.json

```diff
@@ -285,15 +285,15 @@
     "semver": "^5.3.0",
     "shelljs": "^0.7.7",
     "sinon": "^2.2.0",
-    "sri-stats-webpack-plugin": "^0.7.3",
+    "sri-stats-webpack-plugin": "github:sopherio/sri-stats-webpack-plugin#f23de19e1d84dab50e59076e05bcafa508ba64c4",
```

Comment:
This is pretty unfortunate since the patch to make it webpack 2 compatible is just a dependency bump. More info: https://github.com/mikechau/sri-stats-webpack-plugin/issues/71